chia/rpc/full_node_rpc_api.py

```diff
@@ -490,10 +490,6 @@ async def push_tx(self, request: Dict) -> Optional[Dict]:
         if "spend_bundle" not in request:
             raise ValueError("Spend bundle not in request")
 
```

Comment:
I think what you want to do is look for an *absence* of the field you're expecting (namely `coin_spends`), and if it's missing, populate it with whatever's in `coin_solutions` (and include a warning in the response about deprecated API).

Reply:
If we do that, the `from_json_dict` method won't work anymore I don't think but I haven't tested.

Reply:
Yeah I tested it out and it throws a `KeyError` because there is now a "coin_spends" property in the dict